a4-speakup/assets/scss/_variables.scss

```diff
@@ -261,8 +261,8 @@ $line-height-base:            1.5 !default;
 $h1-font-size:                $font-size-base * 3.25 !default;
 $h2-font-size:                $font-size-base * 2.5 !default;
 $h3-font-size:                $font-size-base * 1.75 !default;
-$h4-font-size:                $font-size-base !default;
-$h5-font-size:                $font-size-base * 1.25 !default;
+$h4-font-size:                $font-size-base * 1.25 !default;
+$h5-font-size:                $font-size-base !default;
```

Comment:
@fuzzylogic2000 I don't know why h5 was bigger then h4, do you?

Reply:
ok it's to do with landing page, i'll re-do and add a comment on it so we know for next time